packages/server/src/fhir/operations/execute.ts

```diff
@@ -534,14 +532,16 @@ async function addBotSecrets(
   }
 }
 
+const MIRRORED_CONTENT_TYPES: string[] = [ContentType.FHIR_JSON, ContentType.TEXT, ContentType.HL7_V2];
```

Comment:
I would be in favor or removing FHIR_JSON from this list so that we never automatically use FHIR stringify on Bot responses; especially since FHIR stringify wouldn't have even been used up until about a week ago.

Reply:
I think this makes sense...

What do we think @mattwiller @codyebberson ?

Reply:
That would cause some Bots' output to change from `application/fhir+json` to `application/json`, right?  I think that should be a reasonable change, since both MIME types are compatible